debian/control

```diff
@@ -0,0 +1,15 @@
+Source: neofs-node
+Section: neofs
+Priority: optional
+Maintainer: NeoSPCC <tech@nspcc.ru>
```

Comment:
`NeoSPCC` is this correct?

Reply:
Yes,  NeoSPCC addresses and urls should be used.